GOVERNANCE.md

```diff
@@ -0,0 +1,221 @@
+# cert-manager Governance
+
+This document defines project governance for the cert-manager project. Its
+purpose is to describe how decisions are made on the project and how anyone can
+influence these decisions. We have six levels of responsability, each one
+building on the previous:
+
+- Contributor,
+- GitHub Member,
+- Reviewer,
+- Approver,
+- Maintainer,
+- Admin.
+
+## Contributors
+
+cert-manager is for everyone. Anyone can become a cert-manager contributor
+simply by contributing to the project, whether through code, documentation, blog
+posts, community management, or other means. As with all cert-manager community
+members, contributors are expected to follow the [cert-manager Code of
+Conduct][coc].
+
+All contributions to cert-manager code, documentation, or other components in
+the cert-manager GitHub org must follow the guidelines in [the contributing
+page][contrib]. Whether these contributions are merged into the project is the
+prerogative of the reviewers, approvers and/or maintainers.
+
+## GitHub Members
+
+GitHub Members are active contributors to the cert-manager project.
+
+A contributor is considered to be active when they have had at least one
+interaction (comment on an issue or PR or message in the #cert-manager or #cert-
+manager-dev channels) within the last 18 months.
+
+Members that have been inactive over the past 18 months may be removed from the
+GitHub organization.
+
+**Defined by:** Member of the cert-manager GitHub organization.
+
+### Becoming a GitHub Member
+
+To be added as a GitHub member of the cert-manager organization, you will need
+to look for two sponsors with at least the `reviewer` role. These two sponsors
+must have had some meaningful interaction with you on an issue on GitHub or on
+the cert-manager or cert-manager-dev channels on Slack.
+
+Then, open an issue on the cert-manager repository and mention your sponsors as
```

Comment:
Could it make sense to create a dedicated project for the "community" - as we have in Kubernetes: https://github.com/kubernetes/community. IMO that would be a bit cleaner, and creating a new "container" for the cert-manager community might become useful in the future.

I think it would also make sense to create an issue template for membership request issues.

Reply:
@maelvls I think this makes sense, WDYT?

Reply:
Yes, it makes a lot of sense. @SgtCoDFish is also in favor of having a separate repository for the governance document and the membership GitHub issues and PRs. I'll try to have it sorted today.